langchain-core/src/tracers/tracer_langchain.ts

```diff
@@ -132,62 +108,30 @@ export class LangChainTracer
       if (!current || visited.has(current.id)) continue;
       visited.add(current.id);
 
-      // @ts-expect-error Types of property 'events' are incompatible.
-      this.runMap.set(current.id, current);
+      this.runTreeMap.set(current.id, current);
       if (current.child_runs) {
         queue.push(...current.child_runs);
       }
     }
 
     this.client = runTree.client ?? this.client;
+    this.replicas = runTree.replicas ?? this.replicas;
     this.projectName = runTree.project_name ?? this.projectName;
     this.exampleId = runTree.reference_example_id ?? this.exampleId;
   }
 
-  convertToRunTree(id: string): RunTree | undefined {
```

Comment:
Removing this is technically breaking but would be unusual if someone were extending a our tracer